src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -179,6 +179,11 @@ public ResponseEntity<Exam> updateExam(@PathVariable Long courseId, @RequestBody
 
         if (updatedExam.isMonitoring()) {
             examMonitoringScheduleService.scheduleExamActivitySave(result.getId());
+            examMonitoringScheduleService.notifyMonitoringUpdate(result.getId(), true);
```

Comment:
You could extract it below the if-else and use updatedExam.isMonitoring() for the boolean parameter